### ISSUE TITLE
User Name and email not available past a certain zoom level or page size.

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#4728]
Expected behaviour

 User name and email should be visible, or at least able to be brought up regardless of page size or zoom level at any point.

https://github.com/ushahidi/platform/assets/8624777/0898d5b0-eff5-4dbf-824c-7760f0bf7bef

---
Created automatically from USH-1129 (https://linear.app/ushahidi/issue/USH-1129/user-name-and-email-not-available-past-a-certain-zoom-level-or-page)

**Comment by Aquilaafuadajo** [comment]
Hi, @Angamanga please I'm an outreachy intern and I would like to work on this issue as my first task 🙏🏽 
I have completed my local setup

Here's is proof: 
<img width="1671" alt="Screenshot 2024-03-04 at 18 34 14" src="https://github.com/ushahidi/platform/assets/46582086/374565f3-934f-4b28-9c67-96df0b4fd969">
<img width="1670" alt="Screenshot 2024-03-04 at 18 34 50" src="https://github.com/ushahidi/platform/assets/46582086/6a346b12-8152-4c87-a787-572c78db4b15">
